.github/workflows/main.yml

```diff
@@ -73,14 +73,6 @@ jobs:
       - run: pip check
         if: ${{ !cancelled() }}
         shell: bash
-      - name: Copyright Check
-        run: |
-          python tools/check_copyright.py -check
-        if: ${{ !cancelled() }}
-        shell: bash
-      - run: make spell
-        if: ${{ !cancelled() }}
-        shell: bash
```

Comment:
I have removed the spelling and copyright check as a temporary measure, as these did not exist in terra and logically fail at the moment. I would prefer to start with a minimal CI and then, if we want to add these back, we can open a PR that:

1. adds the checks back
2. fixes the code to pass the checks

(for this reason I have only deleted them from main.yml but keep the related tools that can be ran locally)

Reply:
Similarly, I have removed the deprecation-related actions.

Reply:
The spelling, copyright, mypy etc all passed when this was in Aqua and was run under the sort of CI in the apps. The combined deprecation is something that we added - as otherwise going over all the logs was tedious. The CI was setup to be like Terra but we added spelling, copyright checking and mypy since we, most likely driven by me, thought them important. Since then Terra now errors on deprecation whereas we chose to gather the info so that a dev could/should fix the warnings but did not fail CI. That could be revisited but is likely to add more hard failures to CI.

My take would be to have things commensurate with apps, since we are the team that is presently dealing with all this so its all self-consistent across these repos and then go from there. I understand getting things working incrementally here though as its harder to go from nothing to everything in one go!